reporter/src/main/scala/com/typesafe/tools/mima/lib/analyze/method/MethodChecker.scala

```diff
@@ -37,7 +37,9 @@ private[analyze] abstract class BaseMethodChecker extends Checker[MemberInfo, Cl
 
 private[analyze] class ClassMethodChecker extends BaseMethodChecker {
   def check(method: MemberInfo, inclazz: ClassInfo): Option[Problem] = {
-    if (method.isDeferred)
+    if (method.nonBridge && method.nonAccessible)
```

Comment:
I'm missing something, as I can't see why we are checking for non bridge methods. Why would it matter if the method is a bridge method?